en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/rejecting-revoked-tokens.md

```diff
@@ -77,3 +78,37 @@ Here are the steps to revoking an Access Token.
 
         {"error_message":"Invalid Credentials","code":"900901","error_description":"Invalid JWT token. Make sure you have provided the correct security credentials"}
     ```
+
+## Revoking an API Key
+
+Here are the steps to revoke an API Key.
+
+1. [Deploy the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-in-choreo-connect).
+
+2. Generate an API Key from API-M by following the steps in [Secure APIs with API Keys]({{base_path}}/design/api-security/api-authentication/secure-apis-using-api-keys).
+
+3. Get an access token to the Control Plane (API-M) by referring to [this section]({{base_path}}/reference/product-apis/devportal-apis/devportal-v2/devportal-v2/#section/Authentication). Remember to include `apim:api_key` as a scope when requesting the access token.
+
+4. Revoke the API Key by calling the following API from control plane.
```

Comment:
shall we add this link as well https://apim.docs.wso2.com/en/latest/reference/product-apis/devportal-apis/devportal-v2/devportal-v2/#tag/API-Keys/paths/~1applications~1{applicationId}~1api-keys~1{keyType}~1revoke/post

Reply:
Mentioned it [in line 103](https://github.com/wso2/docs-apim/pull/5656/files#diff-bb12efcd213792e96909740bd0091575622d3753050f1aaa8350f6ee97047398R103) :)